package.json

```diff
@@ -37,7 +37,9 @@
         "@aws-cdk/lambda-layer-kubectl-v24": "^2.0.202",
         "@aws-cdk/lambda-layer-kubectl-v25": "^2.0.3",
         "@aws-cdk/lambda-layer-kubectl-v26": "^2.0.0",
+        "@aws-quickstart/eks-blueprints": "^1.9.1",
```

Comment:
@arpitn2020 please remove changes from package.json. These are only needed for the patterns repo. This repo is the blueprints itself, so it cannot depend on the blueprints or on the addon.

Reply:
removed